packages/launch/package.json

```diff
@@ -30,10 +30,11 @@
 		"watch": "tsc --project ./tsconfig.json --watch"
 	},
 	"dependencies": {
-		"@automattic/react-i18n": "^1.0.0-alpha.0",
+		"@automattic/calypso-analytics": "^1.0.0-alpha.1",
```

Comment:
We should probably also link to the `calypso-analytics` package in `packages/launch/tsconfig.json`, under the `references` key

Reply:
Done